tests/test_infer/test_llama_infer.py

```diff
@@ -53,22 +54,24 @@ def init_to_get_rotary(self, base=10000):
     ],
 )
 def run_llama_test(test_config):
-    sub_model_zoo = model_zoo.get_sub_registry("transformers_llama_for_casual_lm")
-    for name, (model_fn, data_gen_fn, _, _, _) in sub_model_zoo.items():
-        orig_model = model_fn()
-        init_to_get_rotary(orig_model.model, base=10000)
-        orig_model = orig_model.half()
-        data = data_gen_fn()
-
-        shard_config = ShardConfig(
-            enable_tensor_parallelism=True if test_config["tp_size"] > 1 else False, inference_only=True
-        )
-        infer_engine = TPInferEngine(orig_model, shard_config, BATCH_SIZE, MAX_INPUT_LEN, MAX_OUTPUT_LEN)
-
-        generate_kwargs = dict(do_sample=False)
-        outputs = infer_engine.generate(data, **generate_kwargs)
-
-        assert outputs is not None
+    llama_config = LlamaConfig(num_hidden_layers=2, bos_token_id=0, eos_token_id=1, vocab_size=1200, hidden_size=1024)
```

Comment:
why changed like this? no use model zoo?

Reply:
model zoo can't pass the test.